lib/MessageDecoder.ts

```diff
@@ -15,6 +15,9 @@ export class MessageDecoder {
 
     this.registerPlugin(new Plugins.Label_ColonComma(this));
     this.registerPlugin(new Plugins.Label_5Z(this));
+    this.registerPlugin(new Plugins.Label_10_LDR(this));
```

Comment:
Do we have some idea of what LDR is? I think we might want to start putting comments at the very top of each decoder to give some sense of what it is for. Brief summary.

Reply:
no idea - If i did, i'd name it something better than "position report"